manifests/image-references

```diff
@@ -0,0 +1,12 @@
+kind: ImageStream
+apiVersion: image.openshift.io/v1
+spec:
+  tags:
+  - name: cluster-openshift-controller-manager-operator
+    from:
+      kind: DockerImage
+      name: docker.io/openshift/origin-cluster-openshift-controller-manager-operator:latest
```

Comment:
latest? Don't we version them?

Reply:
Do we still push to docker.io?

Reply:
Thanks, done

Reply:
I will do this for the other three operators as well